Dockerfile

```diff
@@ -241,6 +241,14 @@ RUN kubectl krew install resource-capacity && \
     # Install community-images using krew https://github.com/kubernetes-sigs/community-images#kubectl-community-images
     kubectl krew install community-images
 
+# Install pluto: https://github.com/FairwindsOps/pluto
+ENV PLUTO_VERSION 5.18.4
+RUN wget -q "https://github.com/FairwindsOps/pluto/releases/download/v${PLUTO_VERSION}/pluto_${PLUTO_VERSION}_linux_${TARGETARCH}.tar.gz" -O pluto_${PLUTO_VERSION}_linux_${TARGETARCH}.tar.gz && \
+    tar -xvf pluto_${PLUTO_VERSION}_linux_${TARGETARCH}.tar.gz && \
+    rm pluto_${PLUTO_VERSION}_linux_${TARGETARCH}.tar.gz && \
+    chmod +x pluto && \
+    mv pluto /usr/local/bin/pluto
```

Comment:
Please add the completion for the `bash` shell.

Reply:
Done